apps/server/package.json

```diff
@@ -18,27 +18,27 @@
     "test": "catch-uncommitted --skip-node-versionbot-changes && npm run lint && npm run test:integration",
     "doc": "typedoc lib/",
     "prepack": "npm run build",
-    "start": "node --abort-on-uncaught-exception --stack-trace-limit=100 build/index.js",
-    "dev": "NODE_ENV=development nodemon --inspect=0.0.0.0 ./lib/index.ts"
```

Comment:
@karaxuna I'd like to keep `npm run dev`. Could we keep `npm start` and `npm run dev` as-is and call them from `run.sh`? My reason is keeping the ablity to run services locally (not in balena Livepush, but on local PC directly).